src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] inputArray = readingFromFile(fromFileName).split(",");
+        writingInToNewFile(toFileName, inputArray);
+    }
+
+    private static String readingFromFile(String fromFileName) {
```

Comment:
1. Please explain why is this method static.
2. Please use proper naming convention for methods e.g. `readFromFile()`